src/components/WebexLocalMedia/WebexLocalMedia.jsx

```diff
@@ -54,11 +56,12 @@ export default function WebexLocalMedia({
     'no-media': !stream,
   });
   let content;
+  const showBanner = mediaType === 'video' && bannerContent.length;
 
   if (stream) {
     content = (
       <>
-        {mediaType === 'video' && <Banner className="my-preview">My preview</Banner>}
+        {showBanner && <Banner className="my-preview">{bannerContent}</Banner>}
```

Comment:
I think just `{banner && <Banner .../>}` would work just fine, if someone wants a banner for 'screen' we can display it.